12/umbraco-forms/editor/defining-and-attaching-prevaluesources/prevalue-source-types.md

```diff
@@ -17,7 +17,7 @@ Here is a quick overview of them:
 Additional settings can be applied:
 
   * Select which **Value field** should be used for the value of the prevalue.
-  * Select **Use current page as root** instead of choosing a specific root node. Note that the preview is not available when this setting is enabled.
+  * Select **Use current page as root** instead of choosing a specific root node. The preview is not available when this setting is enabled.
```

Comment:
```suggestion
  * Select **Use current page as root** instead of choosing a specific root node. _The preview is not available when this setting is enabled._
```